### ISSUE TITLE
Alliance Approve/Decline Logging

### ISSUE THREAD

**Issue by theonlytruebigmac** [NotEnoughTina/FirstLady#4]
There is an applicant log but would it be possible to include the alliance tag and/or username and time of the acceptance in logs?

**Comment by NotEnoughTina** [comment]
This would depend on the Username / Alliance detection. Currently, it is still not implemented but I have some time this week so I may put some work in on it.

**Comment by theonlytruebigmac** [comment]
I'm also taking a look at it, and I wish tesseract was more accurate lol

**Comment by NotEnoughTina** [comment]
> I'm also taking a look at it, and I wish tesseract was more accurate lol

Yes, I know it's possible, but there's still a lot to learn. The latest revision includes functional alliance detection. While there are still a few misreads, I adjusted the text to align with the detected values so the whitelist feature works properly (e.g., 'S8UP' was changed to 'SSUP').
